roofit/roofit/src/RooNDKeysPdf.cxx

```diff
@@ -1033,36 +1036,53 @@ void RooNDKeysPdf::loopRange(vector<double>& x, map<Int_t,bool>& ibMap) const
    for (Int_t j = 0; j < _nDim; j++) {
       xRm[j] -= _nSigma * (_n * (*_sigmaR)[j]);
       xRp[j] += _nSigma * (_n * (*_sigmaR)[j]);
-      // cout<<"xRm["<<j<<"]="<<xRm[j]<<endl;
-      // cout<<"xRp["<<j<<"]="<<xRp[j]<<endl;
   }
 
-  vector<TVectorD> xvecRm(1,xRm);
-  vector<TVectorD> xvecRp(1,xRp);
+  std::vector<TVectorD> xvecRm(1,xRm);
```

Comment:
I would probably not use TVectorD, is it really needed ?

Reply:
Yes, because the type needs to be matched with the `_sortTVIdcs` data member. If one day we need to optimize the class further, it makes sense indeed to unroll the indices here in a flat vector :+1: